docusaurus/docs/protocol/upgrades/upgrade_list.md

```diff
@@ -18,21 +20,26 @@ The tables below provide a list of past and upcoming protocol upgrades. For more
 - ❓ - Unknown/To Be Determined
 - ⚠️ - Warning/Caution Required
 
-## TestNet
+## MainNet
+Coming...
+
+## Beta TestNet
+Expected to launch on November 21, 2024.
 
+## Alpha TestNet
 :::warning
-This table is currently incomplete and does not include all protocol upgrades. Our recent TestNet upgrades, which were performed via a regenesis, are not listed here.
+Some manual steps are currently required to sync to the latest block. Please follow instructions below.
 :::
 
-<!-- DEVELOPER: if important information about the release is changing (e.g. upgrade hight is changed) - make sure to update the information in GitHub relase as well. -->
+<!-- DEVELOPER: if important information about the release is changing (e.g. upgrade height is changed) - make sure to update the information in GitHub release as well. -->
 
-| Version                                                                  | Planned | Breaking |    Requires Manual Intervention     | Upgrade Height |
-| ------------------------------------------------------------------------ | :-----: | :------: | :---------------------------------: | -------------- |
-| [`v0.0.7`](https://github.com/pokt-network/poktroll/releases/tag/v0.0.7) |    ❓    |    ❓     | ✅ (Alpha TestNet Participants Only) | ❓              |
-| [`v0.0.6`](https://github.com/pokt-network/poktroll/releases/tag/v0.0.6) |    ❓    |    ❓     | ✅ (Alpha TestNet Participants Only) | ❓              |
-| [`v0.0.5`](https://github.com/pokt-network/poktroll/releases/tag/v0.0.5) |    ❓    |    ❓     | ✅ (Alpha TestNet Participants Only) | ❓              |
-| [`v0.0.4`](https://github.com/pokt-network/poktroll/releases/tag/v0.0.4) |    ❓    |    ❓     | ✅ (Alpha TestNet Participants Only) | ❓              |
+| Version                                                                      | Planned | Breaking |                                                          Requires Manual Intervention                                                          | Upgrade Height                                                                                                               |
+| ---------------------------------------------------------------------------- | :-----: | :------: | :--------------------------------------------------------------------------------------------------------------------------------------------: | ---------------------------------------------------------------------------------------------------------------------------- |
+| [`v0.0.10`](https://github.com/pokt-network/poktroll/releases/tag/v0.0.10)   |    ✅    |    ✅     |                                                             ❌ (automatic upgrade)                                                              | ❓ [56860](https://shannon.testnet.pokt.network/poktroll/tx/4E201E5C397AB881F417266154C907D38404BE00BE9A443DE28E44A2B09C5CFB) |
```

Comment:
Why is there a :question: in the "height" column for this row?

Reply:
Good call!